Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/EditorTransformComponentSelection.cpp

```diff
@@ -1728,34 +1728,6 @@ namespace AzToolsFramework
         }
     }
 
-    void EditorTransformComponentSelection::DeselectEntities()
```

Comment:
Looks like this method was just moved? Probably better not to move unless needed to make reviews a little clearer, but not a big deal.

Reply:
Updated :)